server/api/controller/cron.js

```diff
@@ -366,33 +366,31 @@ const fetchIplan = iPlan =>
 					) {
 						plan.set('sent', oldPlan ? 1 : 0);
 					}
-					return plan;
```

Comment:
no need to return the plan?

Reply:
no as it went to the next then which i deleted cause it was redundant